Server/User/routers/users.py

```
@@ -615,6 +615,38 @@ def list_my_friends(
     return [FriendResponse(id=u.id, email=u.email, first_name=u.first_name, last_name=u.last_name) for u in friends]
 
 
+@router.get("/{user_id}/friends", response_model=List[FriendResponse])
+def get_user_friends(
+    user_id: int,
+    db: Session = Depends(get_db),
```

Ruff (0.14.4)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)